plugin-flex-ts-template-v2/src/feature-library/teams-view-enhancements/config.ts

```diff
@@ -10,17 +10,28 @@ const {
 } = (getFeatureFlags()?.features?.teams_view_enhancements as TeamsViewEnhancementsConfig) || {};
 
 const {
+  calls = true,
+  other_tasks = true,
   team = true,
   department = false,
   location = false,
   agent_skills = true,
+  activity_icon = true,
```

Comment:
This goes with the earlier comment to disable the icon column by default.

Reply:
done